### ISSUE TITLE
5N is showing all keycards and asking for king by cuebid and not king count

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#538]
0B8F290C7A0C41DD84C3FA30A251
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/5b591a70-a7b7-4187-94a4-e66983ed0803)

So playing with GIB this is the result

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/0f930989-0f24-46a3-9923-141921ce9d97)

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/738f6ae3-9c47-4d9d-bd7e-4a3c61e3480c)

**Comment by ThePokerDude** [comment]
seems fixed in 8120

![image](https://github.com/EdwardPiwowar/BBA/assets/33804161/2965990c-5344-45f3-a62f-f845f5937941)

**Comment by ThorvaldAagaard** [comment]
No, 6D is wrong, we should not show count, but actual Kings, so the right bid is 6C

**Comment by EdwardPiwowar** [comment]
Later we will add an option to show the king.
